crates/adapters/okx/src/http/client.rs

```diff
@@ -841,166 +881,331 @@ impl OKXHttpClient {
         Ok(trades)
     }
 
-    /// Requests historical candlestick bars for the `instrument_id` and `start` -> `end` time range.
+    /// Requests historical bars for the given bar type and time range.
+    ///
+    /// # Arguments
```

Comment:
Lets remove the argument docs.

For Rust, we can just document anything interesting or specific from line 3 onward in the doc comment.